src/components/ClaimForm.js

```diff
@@ -178,6 +178,19 @@ class ClaimForm extends Component {
       let items = [];
       if (!!this.state.claim.items) {
         items = [...this.state.claim.items];
+
+        let isUnderMaximumAmount = true;
+        
+        items.forEach(item => {
+          if (parseFloat(item.qtyProvided) > (item.item ? (item.item.maximumAmount ? parseFloat(item.item.maximumAmount) : 0) : 0)) {
+            isUnderMaximumAmount = false;
+          }
+        });
+
+        if (!isUnderMaximumAmount) {
+          return false;
+        } 
```

Comment:
You said that you've tried to resolve this using .every function. Are you sure that something like this won't work?
`        const isUnderMaximumAmount = items.every(({ item, qtyProvided }) => {
          const maximumAmount = parseFloat(item?.maximumAmount) || 0;
          const qty = parseFloat(qtyProvided) || 0;
          return qty <= maximumAmount;
        });
        if (!isUnderMaximumAmount) return false;`

Reply:
Because for me using '?' several times in a single line does not look clear. Could you please test provided code and refer to this?